inference-engine/src/transformations/include/ngraph_ops/nms_static_shape_ie.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <vector>
+#include <algorithm>
+#include <string>
+
+#include <transformations_visibility.hpp>
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph {
+namespace op {
+namespace internal {
+
+template <typename BaseNmsOp>
+class NmsStaticShapeIE : public BaseNmsOp {
+public:
+    NGRAPH_RTTI_DECLARATION;
+
+    using Attributes = typename BaseNmsOp::Attributes;
+
+    /// \brief Constructs a NmsStaticShapeIE operation
+    ///
+    /// \param boxes Node producing the box coordinates
+    /// \param scores Node producing the box scores
+    /// \param attrs Attributes of the operation
+    NmsStaticShapeIE(const Output<Node>& boxes,
+                     const Output<Node>& scores,
+                     const Attributes& attrs) : BaseNmsOp(boxes, scores, attrs) {
+        validate_and_infer_types();
```

Comment:
Nitpick: in the constructor, we usually use ` constructor_validate_and_infer_types();`

Reply:
Updated.